default/stringtables/en.txt

```diff
@@ -6832,6 +6844,9 @@ VICTORY! The %empire% has transcended our reality and wins a technological victo
 VICTORY_ALL_ENEMIES_ELIMINATED
 VICTORY! The %empire% is the last surviving empire.
 
+VICTORY_FEW_HUMANS_ALIVE
+VICTORY! The %empire% is one of the last surviving empire ruled by humans.
```

Comment:
ruled by humans -> controlled by a human player

"human" is an in-game species, so this is misleading as written.

Reply:
Fixed.